packages/syntax-core/src/Button/Button.module.css

```diff
@@ -98,6 +98,7 @@
 }
 
 .textContainer {
+  flex-shrink: 0;
```

Comment:
@jliotta instead of setting it on `Button`, can we wrap that button with a `<Box />` which has `flex-shrink: 0`?

Asking because in certain cases, we do want the text to wrap:
![image](https://user-images.githubusercontent.com/127199/233752391-889c8820-789b-4416-af8b-6e8b282ac3a2.png)

Otherwise we are going to get into a situation where we see horizontal scrollbars:

![image](https://user-images.githubusercontent.com/127199/233752452-c300165e-f014-42fa-b900-3159436ef9c7.png)